gluten-flink/planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecCalc.java

```diff
@@ -112,41 +111,40 @@ public Transformation<RowData> translateToPlanInternal(
         final Transformation<RowData> inputTransform =
                 (Transformation<RowData>) inputEdge.translateToPlan(planner);
 
-        Type inputType = LogicalTypeConverter.toVLType(inputEdge.getOutputType());
+        /// These codes are changed for gluten
```

Comment:
Nit:
Recommend to use the below comments to improve readability and visually separate the section.

// --- Begin Gluten-specific code changes ---


// --- End Gluten-specific code changes ---